### PR TITLE
Deposit mobile

### LINKED ISSUE #594: Implement Deposit mobile screen
https://invis.io/CANS9NGWBEJ

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #594.

### CODE REVIEW

File: src/view/deposit-mobile.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  btnWrapper: {
+    alignItems: 'center',
+  },
+  copyBtn: {
+    borderRadius: 0,
+    paddingLeft: 15,
+    paddingRight: 15,
+    backgroundColor: color.glas,
+  },
```
Comment (tanx) [inline src/view/deposit-mobile.js]: We should create another `SmallCopyButton` component that we can reuse in the invoice screen.

File: src/component/button.js [hunk 2]
```diff
@@ -418,6 +418,7 @@ const copyStyles = StyleSheet.create({
   },
   text: {
     fontFamily: 'OpenSans Light',
+    textAlign: 'center',
```
Comment (tanx) [inline src/component/button.js]: ... changing the CopyButton for desktop shouldn't be necessary.

File: src/component/header.js [hunk 3]
```diff
@@ -103,7 +103,7 @@ export const Title = ({ title = '', style, children }) => (
 
 Title.propTypes = {
   title: PropTypes.string,
-  style: View.propTypes.style,
+  style: Text.propTypes.style,
```
Comment (tanx) [inline src/component/header.js]: Good catch 👍

File: src/component/button.js (hunk 2 above)
Comment (valentinewallace) [reply src/component/button.js]: fixed!

File: src/view/deposit-mobile.js (hunk 1 above)
Comment (valentinewallace) [reply src/view/deposit-mobile.js]: Sounds good! should i open an issue for that or did you want it in the same PR?

**Pull request merged**